modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/WebSocketManager.java

```diff
@@ -72,9 +77,36 @@
     /** */
     private static final Logger log = LoggerFactory.getLogger(WebSocketManager.class);
 
+    /** */
+    private static final String VISOR_IGNITE = "org.apache.ignite.internal.visor.";
+
+    /** */
+    protected static final String JAVA_TUPLE = "org.apache.ignite.lang.IgniteBiTuple";
+
+    /** */
+    protected static final String JAVA_STRING = "java.lang.String";
```

Comment:
Why we need this constant? We can use `String.class.getName()`

Reply:
Fixed.